test/inductor/test_aot_inductor.py

```diff
@@ -1516,6 +1537,7 @@ def fail_minimal_arrayref_interface(is_skip=False):
     "test_repeat_output": fail_with_and_without_stack_allocation(is_skip=True),
     # the test segfaults
     "test_scatter_fallback": fail_with_and_without_stack_allocation(is_skip=True),
+    "test_scatter_reduce_fallback": fail_stack_allocation(is_skip=True),
```

Comment:
use `fail_stack_allocation` or else you get this:

Error: cannot convert ‘torch::aot_inductor::ArrayRefTensor<long int>’ to ‘AtenTensorHandle’ {aka ‘AtenTensorOpaque*’}
  592 |     aoti_torch_scatter_reduce_out(buf0, buf0,0,arg1_1,arg2_1, "sum",1);

Reply:
cc @desertfire.